src/components/FormElements/MuxedIdPicker.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Input, InputProps } from "@stellar/design-system";
+
+interface MuxedIdPickerProps extends Omit<InputProps, "fieldSize"> {
+  id: string;
+  fieldSize?: "sm" | "md" | "lg";
+  label: string;
+  labelSuffix?: string | React.ReactNode;
+  placeholder?: string;
+  readOnly?: boolean;
+  value: string;
+  error: string | undefined;
+  // eslint-disable-next-line no-unused-vars
+  onChange?: (e: React.ChangeEvent<any>) => void;
+}
+
+export const MuxedIdPicker = ({
```

Comment:
:thinking: Do we need this as a separate component? We could use an `Input` instead, passing in a placeholder we need here. What do you think?